internal/service/project/resource_project.go

```diff
@@ -828,3 +799,17 @@ func getChangesInLimitsSet(planLimits, stateLimits []TfLimitModel) (newElements,
 	}
 	return newLimits, changedLimits, removedLimits
 }
+
+func setProjectBool(plan types.Bool, setting **bool) {
+	if !plan.IsUnknown() {
+		*setting = plan.ValueBoolPointer()
+	}
+}
+
+func updateProjectBool(plan, state types.Bool, setting **bool) bool {
+	if plan != state {
+		*setting = plan.ValueBoolPointer()
+		return true
+	}
+	return false
+}
```

Comment:
nice you had this idea of moving to separate method. Can we unit test this now?

Reply:
done